jax/numpy/linalg.py

```diff
@@ -136,6 +136,32 @@ def eigvalsh(a, UPLO='L'):
   return w
 
 
+@_wraps(onp.linalg.pinv)
+def pinv(a, rcond=None):
+  # ported from https://github.com/numpy/numpy/blob/v1.17.0/numpy/linalg/linalg.py#L1890-L1979
+  a = np.conj(a)
+  if rcond is None:
+      if np.ndim(a) < 2:
+        num_rows = num_cols = np.shape(a)[-1]
+      else:
+        num_rows = np.shape(a)[-2]
+        num_cols = np.shape(a)[-1]
+      max_rows_cols = np.max(num_rows, num_cols)
+      rcond = 10. * max_rows_cols * np.finfo(a.dtype)
```

Comment:
You need pull off the appropriate attribute from `finfo`, e.g.,
```suggestion
      rcond = 10. * max_rows_cols * np.finfo(a.dtype).eps
```

I'm actually rather puzzled by what is going on here -- how could your unit tests be possible passing? Are you sure they are actually being run?

Reply:
Ah, i think i messed up the indentation of my test and they weren't being run. They should be running now, but failing on big matrix cases. :(